docs/user-guide/server-side-install-overview.md

```diff
@@ -1,19 +1,122 @@
+    * [Tasks performed by the security administrator](#tasks-performed-by-the-security-administrator)
+    * [Tasks performed by the storage administrator](#tasks-performed-by-the-storage-administrator)
+    * [Tasks performed by the network administrator](#tasks-performed-by-the-network-administrator)
+    * [Tasks performed by the system programmar](#tasks-performed-by-the-system-programmar)
 
-What to be covered in the overview:
+## Zowe server-side components in the installation
 
-End-to-end install flow diagram 
+The following Zowe server-side components are included in this installation: 
 
-config flow diagram
+* **Zowe Application Framework**  
+A web user interface (UI) that provides a virtual desktop containing several apps allowing access to z/OS function.  <details><summary>Click here for more information about Zowe Application Framework</summary> Zowe™ Application Framework is a  Base Zowe includes apps for traditional access such as a 3270 terminal and a VT Terminal, as well as an editor and explorers for working with JES, MVS Data Sets and Unix System Services
+</details>
 
-Roles & Responsibilities
+*  **Zowe API Mediation Layer (API ML)**  
+Zowe API Mediation Layer provides a reverse proxy and enables REST APIs by providing a single point of access for mainframe service REST APIs.<details><summary>Click here for more information about API Mediation Layer.</summary> Such APIs include MVS Data Sets, JES, as well as working with z/OSMF. The API ML has dynamic discovery capability for these services and Gateway is also responsible for generating the authentication token used to provide single sign-on (SSO) functionality.</details>
 
-Zowe Key Terms & Concepts: Zowe runtime, Zowe Application Framework, Zowe API Mediation Layer, Z Secure Services (ZSS), Zowe Cross Memory Server (@Joe)
+* **Zowe System Services (ZSS)**  <!-- Is ZSS installed with Zowe runtime or is is part of ZIS installation in the configuring section? -->
+Working closely with ZIS, ZSS serves as one of the primary, authenticated back-ends that communicates with z/OS and provides Zowe with a number of APIs including z/OS Unix files and data sets, control of the plug-ins and services lifecycle, and security management.<details><summary>Click here for more information about Zowe System Services.</summary> The Zowe Desktop delegates a number of its services to ZSS which it accesses through the default http port `7557`. ZSS is written in C and uses native calls to z/OS to provide its services.</details>
 
-Technology Needs: YAML, node.js, USS background details (existing content)
+## Installation of Zowe runtime
+
+Zowe runtime refers to the full, unarchived set of binaries, executable files, scripts, and other elements that are run when Zowe is started.
+Runtime installation includes the following components:  
+* Zowe Application Framework
+* Zowe API Mediation Layer
+* Z Secure Services (ZSS)
```

Comment:
I thought it standed for Zowe System Services

Reply:
Well the project official name is Zowe System Services. 

Also the things mentioned are Projects and not components. 
The components would be something like:

- API Gateway
- Discovery Service
- API Catalog
- ZSS
And I don't know how Zowe Application Framework is split in components.